### ISSUE TITLE
V1.1.13: repeat setting ignored

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#210]
Time for some minors: I added 4 tracks to the queue, with repeated off selected. This setting was ignored and the queue repeated.

**Comment by marcelveldt** [comment]
Can you have a look at the queue when that happens if the items return ?
Are you sure playback is repeating all 4 songs again ?

**Comment by erkr** [comment]
Yes it does, and it's not specific for 4. I recorded a short movie where this happens with a queue of two tracks. 
https://user-images.githubusercontent.com/4363599/169690460-f6a97050-8eb7-42c6-890a-be216328cae1.MOV

It start showing the first track that was played. Then I tap on the next tab, just before the second track finishes, just in time to see the queue repeating. 

I also tried to toggle the repeat off setting. That doesn't help

**Comment by marcelveldt** [comment]
I have reproduced it. Working on a fix.

**Comment by erkr** [comment]
This issue is basically fixed, as the queue doesn't repeat anymore. 
But it introduced a minor error. When the last track is completed it doesn't move the the 'player items' list.
Ps: I like the change to gray out the active track👍

**Comment by marcelveldt** [comment]
What do you mean ? You mean the queue isn't cleared after the last track finished ?

**Comment by erkr** [comment]
Yes, the last track of the queue stays in the 'play next' tab when finished. As a consequence the queue can not be restarted if I would like to do that. I have te recreate it then.
I think it should nice to the plated items as well, but of course not automatically restart.

**Comment by erkr** [comment]
I saw the loading order is now changed in v1.1.18 👍

**Comment by erkr** [comment]
> I saw the loading order is now changed in v1.1.18 👍

Sorry, reply on the wrong issue

**Comment by erkr** [comment]
Just checked this behavior in v1.1.19: besides the glitch that the last track of the queue stays in the 'play next' tab when finished, I noticed another glitch: 
When repeat is on, the first track started to play again, but the play next list remained empty. Will test a little more.

**Comment by marcelveldt** [comment]
fixed once merged and released
https://github.com/music-assistant/music-assistant-lib/pull/349

**Comment by erkr** [comment]
Yes, fixed. Thanks!
